modules/candidate_list/test/candidate_listTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * candidate_list automated integration tests
+ *
+ * PHP Version 5
+ *
+ * @category Test
+ * @package  Loris
+ * @author   Ted Strauss <ted.strauss@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+
+require_once __DIR__ . "/../../../test/integrationtests/LorisIntegrationTest.class.inc";
+class candidateListTestIntegrationTest extends LorisIntegrationTest
+{
+    /**
+     * Tests that, when loading the candidate_list module, some
+     * text appears in the body.
+     *
+     * @return void
+     */
+    function testCandidateListDoespageLoad()
+    {
+        $this->webDriver->get($this->url . "?test_name=candidate_list");
+        $bodyText = $this->webDriver->findElement(WebDriverBy::cssSelector("body"))->getText();
+        $this->assertContains("Access Profile", $bodyText);
+    }
+}
```

Comment:
Tara's pull request deletes the certification module. I don't think we should include this test.